package.json

```diff
@@ -48,7 +48,7 @@
   "scripts": {
     "build": "npm -s run clean && rollup -c",
     "build:standalone": "run-s build standalone",
-    "check:audit": "npm audit",
+    "check:audit": "npm audit --audit-level=moderate",
```

Comment:
https://app.circleci.com/pipelines/github/marp-team/marp-cli/1604/workflows/a319d270-4aad-4505-baf5-b0d46e32cba2/jobs/7566/parallel-runs/0/steps/0-105

We have already updated the affected module `brace-expansion` into patched versions, but `npm audit` sill fails because the security advisory is not yet updated to reflect new version range excluding patched releases.

This change is a workaround for pull requests that are waiting to merge.